### PR TITLE
feat: discover finish

### PR DESCRIPTION
**Pull request by Samet91**

### CODE REVIEW

**Review by iva585 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -1,64 +1,65 @@
...
+            src="images/woman_desktop.png" 
             alt="Woman rubbing her hands with chalk before workout"
           /><br />
         </div>
         <div class="landingpage__title">
           <span class="landingpage__title--big">4 WEEK</span>
           <h1>WORKOUT PLAN</h1>
         </div>
-        <p class="landingpage__text">
+        <p class="paragraph">
```
Comment (F-Kirchhoff) [inline index.html:62]: I would stick with the BEM naming convention, even if we use paragraph in another part. I think its more clean if we have two paragraphs for each block.

File: index.html [hunk 2]
```diff
@@ -83,7 +92,19 @@ <h3 class="training__img-subtitle">YOGA</h3>
         </div>
         <a class="workout-button" href="#">SEE MORE</a>
       </section>
-      
+
+      <section class="discover-section">
+        <h2 class="discover__title">DISCOVER YOUR POTENTIAL</h2>
+        <p class="paragraph">
```
Comment (F-Kirchhoff) [inline index.html:98]: here then use discover__text

File: index.html [hunk 3]
```diff
@@ -83,7 +92,19 @@ <h3 class="training__img-subtitle">YOGA</h3>
...
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Quidem nemo
+          officiis a nam corporis quisquam provident assumenda odio ut
+          consequatur.
+        </p>
+        <div class="discover__align-buttons">
```
Comment (F-Kirchhoff) [inline index.html:103]: discover__button-container would be more common and better to understand for the average programmer

File: index.html [hunk 4]
```diff
@@ -83,7 +92,19 @@ <h3 class="training__img-subtitle">YOGA</h3>
...
+            <a class="workout-button" href="#">VIEW PLANS</a>
+            <a class="workout-button" href="#">LEARN MORE</a>
```
Comment (F-Kirchhoff) [inline index.html:105]: second button gets a --modifier like workout-button--dark

Comment (F-Kirchhoff) [reply index.html:105]: as a second class additionally to workout-button

File: style/components/discover.css [hunk 5]
```diff
@@ -0,0 +1,19 @@
+.discover-section {
+    background-color: var(--c-mud);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```
Comment (F-Kirchhoff) [inline style/components/discover.css:5]: nice!

File: style/components/discover.css [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    margin: 30px 0;
```
Comment (F-Kirchhoff) [inline style/components/discover.css:6]: lets make the margin via a general base-class css for all sections

File: style/components/discover.css [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    padding: 25px;
+}
+.discover__title {
+    font-size: 2.5rem;
+    text-align: center;
+    line-height: 3rem;
```
Comment (F-Kirchhoff) [inline style/components/discover.css:12]: Do you want the height to change when someone increases the rem size?

File: style/components/landingpage.css [hunk 8]
```diff
@@ -2,13 +2,25 @@
...
+  
+  
 }
 
 .img_1__frame {
   margin-top: 100px;
-  overflow: hidden;
   border-radius: 20px;
   overflow: hidden;
+  width: 90%;
```
Comment (F-Kirchhoff) [inline style/components/landingpage.css:13]: give the frame a margin than set it to a % width

File: style/components/landingpage.css [hunk 9]
```diff
@@ -2,13 +2,25 @@
...
+  height: 350px;
+  position: relative;
+}
+
+.img_1 {
+  display: block;
+  position: absolute;
+  width: 170%;
```
Comment (F-Kirchhoff) [inline style/components/landingpage.css:21]: dont do that

File: style/components/landingpage.css [hunk 10]
```diff
@@ -26,12 +38,6 @@
   margin-bottom: -15px;
 }
 
-.landingpage__text {
-  color: var(--c-white);
```
Comment (F-Kirchhoff) [inline style/components/landingpage.css:30]: we have set the default color of body to this var, so we dont need it here

File: style/components/paragraph.css [hunk 11]
```diff
@@ -0,0 +1,7 @@
+
+.paragraph {
+    color: var(--c-white-faded);
```
Comment (F-Kirchhoff) [inline style/components/paragraph.css:3]: maybe stay with the var white and give it some opactiy of 0.8?

File: style/components/training.css [hunk 12]
```diff
@@ -3,6 +3,7 @@
   display: flex;
   flex-direction: column;
   align-items: center;
+  width: 90%;
```
Comment (F-Kirchhoff) [inline style/components/training.css:6]: better: some fixed margin and width: 100%

File: style/components/workout-button.css [hunk 13]
```diff
@@ -1,7 +1,9 @@
 .workout-button {
     color: var(--c-white);
-    background-color: hotpink;
+    background-color: var(--c-pink);
```
Comment (F-Kirchhoff) [inline style/components/workout-button.css:3]: good!

File: style/components/workout-button.css [hunk 14]
```diff
@@ -1,7 +1,9 @@
 .workout-button {
     color: var(--c-white);
```
Comment (F-Kirchhoff) [inline style/components/workout-button.css:2]: not needed see above

File: style/components/workout-button.css [hunk 15]
```diff
@@ -1,7 +1,9 @@
...
-    background-color: hotpink;
+    background-color: var(--c-pink);
     text-decoration: none;
     padding: 10px;
-    border-radius: 10px;
+    border-radius: 6px;
+    padding-left: 20px;
```
Comment (F-Kirchhoff) [inline style/components/workout-button.css:7]: use padding: 0 20px;

**Pull request merged**